redfish-core/include/error_messages.hpp

```diff
@@ -608,6 +608,17 @@ nlohmann::json sourceDoesNotSupportProtocol(const std::string& arg1,
 void sourceDoesNotSupportProtocol(crow::Response& res, const std::string& arg1,
                                   const std::string& arg2);
 
+/**
+ * @brief Formats StrictAccountTypes message into JSON
+ * Message body: Indicates the request failed because a set of `AccountTypes` or
+ * `OEMAccountTypes` was not accepted while `StrictAccountTypes` is set to `true
+ * @param[in] arg1 Parameter of message that will replace %1 in its body.
```

Comment:
Maybe rephrase? 

What does %1 mean? I don't see %1 in error_message.cpp definitions of strictAccountTypes().

Reply:
%1 got replaced with arg1 in that code